tests/test_time.py

```diff
@@ -24,7 +24,8 @@
 ONE_DAY = 24 * ONE_HOUR
 ONE_YEAR = 365.25 * ONE_DAY
 
-with freeze_time("2020-02-02"):
+FROZEN_DATE = "2010-02-02"
```

Comment:
(GitHub wouldn't let me do it as a single suggestion.)
```suggestion
FROZEN_DATE = "2010-02-02"

```